packages/react-scripts/config/webpack.development.js

```diff
@@ -0,0 +1,62 @@
+// @remove-on-eject-begin
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @remove-on-eject-end
+'use strict';
+
+const path = require('path');
+const paths = require('./paths');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CaseSensitivePathsPlugin = require('case-sensitive-paths-webpack-plugin');
+const WatchMissingNodeModulesPlugin = require('react-dev-utils/WatchMissingNodeModulesPlugin');
+
+// We are in the development config. This contains development only features.
+module.exports = function() {
+  return {
+    devtool: 'cheap-module-source-map',
+    // Note: instead of the default WebpackDevServer client, we use a custom one
+    // to bring better experience for Create React App users. You can replace
+    // the line below with these two lines if you prefer the stock client:
+    // require.resolve('webpack-dev-server/client') + '?/',
+    // require.resolve('webpack/hot/dev-server'),
+    entry: [
+      require.resolve('react-dev-utils/webpackHotDevClient'),
+      paths.appIndexJs,
+    ],
+    output: {
+      pathinfo: true,
+      filename: 'static/js/bundle.js',
+      chunkFilename: 'static/js/[name].chunk.js',
+      devtoolModuleFilenameTemplate: info =>
+        path.resolve(info.absoluteResourcePath).replace(/\\/g, '/'),
+    },
+    plugins: [
+      new HtmlWebpackPlugin(
+        Object.assign(
+          {},
+          {
+            inject: true,
+            template: paths.appHtml,
+          }
+        )
+      ),
+      new webpack.ProgressPlugin(),

```

Comment:
There's been lots of discussion in the past about different progress plugin issues (inaccuracy, slowness) (eg. back in https://github.com/facebook/create-react-app/pull/1011, last time in https://github.com/facebook/create-react-app/issues/6310#issuecomment-474977767). How's the current state of the `webpack.ProgressPlugin` compared to those old solutions?

Reply:
I don't know what versions of webpack these comments where in context to, however we have made a few enhancements and fixes over webpack 2 and 3.  cc @sokra for his comments.

Reply:
The lastest version has some fixes and ProgressPlugin should have very small impact.

see also https://github.com/webpack/webpack/pull/9466